.travis.yml

```diff
@@ -2,16 +2,25 @@ sudo: required
 dist: trusty
 language: python
 before_install:
-  - node --version
-  - npm install --python=python2 -g dredd
-  - bundle install
+- node --version
+- npm install --python=python2 -g dredd
+- bundle install
 python:
-  - "2.7"
-  - "3.2"
-  - "3.3"
-  - "3.4"
-  - "3.5"
+- '2.7'
+- '3.2'
+- '3.3'
+- '3.4'
+- '3.5'
 script:
-  - python -B -m unittest discover
-  - pip install -e .
-  - bundle exec cucumber
+- python -B -m unittest discover
+- pip install -e .
+- bundle exec cucumber
+deploy:
+  provider: pypi
+  user: wvi
+  password:
+    secure: G2gtzJE0zn/AnS7Z7RPiyTJMV583yJitBg8Un4/yA1mr9eNzEDw7s8EA1CRYMN/y9rizQ/xbtqonqTSjWdXJOu4g0+tXMUC3CZ5BqMxWVaQBQGx7XRF9t0YolhzznMo4i9rtwVAo8Bq0KlOWtjFpbTW2G0PqvIGWgUv/xo/JabWj6YyDmtnciF1KxtfVlD/RoSnCztyOzpqQqr6nQPLK221acELYCAAIE13L/3tJm8ChpGsDW8My3Z3gxQd50rQW7Z79CPxQci3reTTCx45uOOPLqT4oBUH6w1WXy9Aj/yjJAMchlQBS9i1MdAkqCClNlgpdvX928rQy6PkGGTxB/TlcgIXToMr7N/AlfqohVLrkE4iNHi51H3m9+PROzctfrpl3L4pos55ogG9gjeLPOMQaq8w7khHyEcyJMOEfrfuQYYs/enavq7fdFM8Ia92gCwowjLWaYXSsHm4t614a+as0gjmV96NOi9ClFqMDVIgwI8PPFoKuAKwmuus017heTphB9elAeIPWVQGHiF3vz/DorG0Kf6mo8Hq1mJMuO3pFgbEP5PH8e2LBwYofbv7GPdqMgDZ3pzUZeAVZEfKXmATA2JtvutqGJEPvfd79h/idhHQcaXK+W/pzShLck+EOPG9gzcWccpNmlioQ09YvOLyN6CDDF/jz0qDvinu0tHM=
+  on:
+    tags: true
+    distributions: sdist bdist_wheel
```

Comment:
Distributions shouldn't be inside `on` section but inside the `deploy` section.

Ref: https://docs.travis-ci.com/user/deployment/pypi#Uploading-different-distributions